### ISSUE TITLE
Generating subtitles that are too long

### ISSUE THREAD

**Issue by Sir0xb** [shaishaicookie/fcpx-auto-captions#8]
If you create YouTube shorts, the generated subtitles are too long. Can shorter and more detailed generation patterns be supported?

**Comment by user72356** [comment]
I agree and was about to open an issue about this. No word from author, sadly. Capcut generates shorter captions which is great (the only great thing about Capcut).
